src/docs/markdown/api.md

```diff
@@ -90,33 +96,33 @@ Export entire config and pretty-print it:
 
 <pre><code class="cmd"><span class="bash">curl "http://localhost:2019/config/" | jq</span>
 {
-	"apps": {
-		"http": {
-			"servers": {
-				"myserver": {
-					"listen": [
-						":443"
-					],
-					"routes": [
-						{
-							"match": [
-								{
-									"host": [
-										"example.com"
-									]
-								}
-							],
-							"handle": [
-								{
-									"handler": "file_server"
-								}
-							]
-						}
-					]
-				}
-			}
-		}
-	}
+  "apps": {
```

Comment:
Why the change to spaces? Tabs result in a small file and are user-configurable, and less error-prone.

Reply:
Not user configurable in the docs though. Tab size is dictated by our CSS.

https://developer.mozilla.org/en-US/docs/Web/CSS/tab-size

I don't think the file size argument is relevant. Point is, it's easier to read deeply nested JSON with less width for each indentation level (I kinda want to make this change everywhere in the docs frankly)

Reply:
I think this change is very unrelated to v2.5 and should have separate discussion first. Maybe let's leave this out for consistency sake for now.

Reply:
ok